### ISSUE TITLE
Quit Confirmation is unreadable

### ISSUE THREAD

**Issue by rzelnik** [johannesjo/super-productivity#3034]
### Your Environment

- Version used: 8.0.0 snap
- Operating System and version: Ubuntu 23.10
- Desktop Environment: Gnome

### Expected Behavior

Quit Confirmation modal window text should be readable.

### Current Behavior

I see some strange squares instead of text.

### Steps to Reproduce (for bugs)

1. Click "window close" button
2. Quit Confirmation dialog appears

### Can you reproduce this reliably?

The broken confirmation modal appears every time I quit the app and with any language in language settings.

### Error Log (Desktop only)

I didn't find any fresh log entries, just some old ones in ~/snap/superproductivity/current/.config/superProductivity/log/main.log

[2023-11-18 12:23:49.672] [info]  close, isQuiting: undefined
[2023-11-18 12:23:49.674] [info]  Actions to wait for  [ 'FINISH_DAY_BEFORE_CLOSE_EFFECT' ]
[2023-11-18 12:23:49.675] [info]  BEFORE_CLOSE_DONE FINISH_DAY_BEFORE_CLOSE_EFFECT []
[2023-11-18 12:23:49.676] [info]  close, isQuiting: undefined
[2023-11-18 12:23:49.677] [info]  close, isQuiting: true
[2023-11-18 12:23:49.686] [info]  Quit after all windows being closed

### Screenshot

![screenshot](https://github.com/johannesjo/super-productivity/assets/610651/1b747822-8777-4fbd-aeff-23f577965d70)

**Comment by johannesjo** [comment]
I am unable to reproduce this unfortunately.

**Comment by rzelnik** [comment]
Ok, I will try to debug this locally.

**Comment by johannesjo** [comment]
Thank you! If there is a way to fix this on our end, it's likely related to electron.

**Comment by rzelnik** [comment]
I already have local dev environment running. Is there any way how to activate the quit confirmation modal window? I have "Confirm before exiting the app without finishing day first" turned on, I have unfinished tasks in today's list, but the confirmation modal doesn't appear. Sorry for dumb questions, but I am not skilled in node.js/electron development. I just discovered that the app quits in electron/main-window.ts line 252: `_quitApp();` (with `ids.length == 0`).
